client/app/components/sections/Topbar/Topbar.css

```diff
@@ -1,5 +1,5 @@
-@custom-media --Topbar_small (min-width: 30rem);
-@custom-media --Topbar_medium (min-width: 50rem);
+@custom-media --Topbar_small (min-width: 38em);
+@custom-media --Topbar_medium (min-width: 62em);
```

Comment:
These should be defined in `client/app/assets/styles/media-queries.js` if they are also used in components within Topbar e.g. to switch between mobile and tablet/desktop layouts.